lib/src/rules/use_rethrow_when_possible.dart

```diff
@@ -62,6 +62,8 @@ class _Visitor extends SimpleAstVisitor<void> {
 
   @override
   void visitThrowExpression(ThrowExpression node) {
+    if (node.parent is ConditionalExpression) return;
```

Comment:
Should this just be `is Expression`? Consider
```
try {
} catch (e) {
  o.m(throw e);
}
```
It's poor code, but I believe that this would also be a false positive.

Reply:
Oh! I never thought this could even be possible.

Thanks for hinting.